apps/backend/sample-files/transaction.json

```diff
@@ -1,176 +1,173 @@
 [
-    {
-      "transaction_id": "72cbe1da-b08e-49d4-9a87-69afd660d14b",
-      "transaction_date": "2023-01-01",
-      "transaction_time": "13.33.31.875973",
-      "fiscal_close_date": "2023-01-02",
-      "amount": 25.20,
-      "void_indicator": false,
-      "miscellaneous": {
-        "employee_id": "SC61350"
+  {
+    "transaction_id": "72cbe1da-b08e-49d4-9a87-69afd660d14b",
+    "transaction_date": "2023-01-01",
+    "transaction_time": "13.33.31.875973",
+    "fiscal_close_date": "2023-01-02",
+    "total_payment_amount": 25.2,
+    "void_indicator": false,
+    "miscellaneous": {
+      "employee_id": "SC61350"
+    },
+    "source": {
+      "source_id": "SBC",
+      "location_id": 61,
+      "accepted_payment_methods": ["CASH", "CHQ", "P", "M", "V"]
+    },
+    "payments": [
+      {
+        "amount": 10.2,
+        "currency": "CAD",
+        "exchange_rate": 0,
+        "payment_method": "CASH",
+        "payment_channel": "in-person"
       },
-      "source": {
-        "source_id": "SBC",
-        "location_id": 61,
-        "accepted_payment_methods": ["CASH", "CHQ", "P", "M", "V"]
-      },
-      "payments": [
-        {
-          "amount": 10.20,
-          "currency": "CAD",
-          "exchange_rate": 0,
-          "payment_method": "CASH",
-          "payment_channel": "in-person"
+      {
+        "amount": 15.0,
+        "currency": "CAD",
+        "exchange_rate": 0,
+        "payment_method": "V",
+        "payment_channel": "in-person",
+        "terminal": {
+          "card_no": "5253",
+          "merchant_id": "20777441",
+          "device_id": "GA2077744108",
+          "invoice_no": ""
         },
-        {
-          "amount": 15.00,
-          "currency": "CAD",
-          "exchange_rate": 0,
-          "payment_method": "V",
-          "payment_channel": "in-person",
-          "terminal": {
-            "card_no": "5253",
-            "merchant_id": "20777441",
-            "device_id": "GA2077744108",
-            "invoice_no": ""
-          },
-          "online": {
-            "tran_id": "",
-            "order_no": ""
-          },
-          "pos": {
-            "approval_code": "ASWQD24342"
-          }
+        "online": {
+          "tran_id": "",
+          "order_no": ""
+        },
+        "pos": {
+          "approval_code": "ASWQD24342"
         }
-      ],
-      "accounting": {
-        "items": [
+      }
+    ],
+    "accounting": [
+      {
```

Comment:
Please note that I have removed the key of "items" from our example in order to align with the current Transaction DTO